src/Symfony/Bridge/Twig/Resources/views/Form/bootstrap_3_layout.html.twig

```diff
@@ -123,10 +123,15 @@
 {%- endblock datetime_row %}
 
 {% block checkbox_row -%}
+{% spaceless %}
```

Comment:
Is `spaceless` here really needed? It makes the block much slower and we removed them everywhere when not needed. I can see some more usages, but we should consider removing them if not strictly needed.

Reply:
you're right, it's not needed, but `spaceless` was there when the template was added back in 2014 and got copy-pasted over the time for other row-blocks. 

to make sure this gets merged asap and to avoid possible BC breaks, i only re-added the old code with `spaceless`.

but... after reviewing the code myself today, i noticed that i made a mistake:

the code should've been re-added to the `bootstrap_3_horizontal_layout.html.twig` (where it was removed) instead of replacing the `checkbox_row`-block in the the `bootstrap_3_layout.html.twig` (it causes an unnecessary empty container in the vertical layout now).

i'll prepare a new PR where i'll fix this when #24727 gets merged to avoid conflicts.